notebooks/official/tensorboard/tensorboard_profiler_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>with prebuilt container -&gt; with a prebuilt container</p><p> </p><p>I would drop the step: Setup a service account and a Cloud Storage bucket</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Changed</p>